lib/livebook/evaluator.ex

```diff
@@ -144,6 +144,7 @@ defmodule Livebook.Evaluator do
       end
 
     Evaluator.IOProxy.flush(state.io_proxy)
+    Evaluator.IOProxy.clear_input_buffers(state.io_proxy)
```

Comment:
I would consider having `flush` doing both operations but it is your call. :)

Reply:
Wanted to do so initially, but I think they are quite different in semantics. It's perfectly fine to call `flush` at any time, but that's not the case for the inputs.

Reply:
Ah, makes sense :+1: